docs/content/users/install/performance.md

```diff
@@ -137,6 +137,7 @@ Mutagen can offer a big performance boost on macOS and Windows. It’s fast and
 
     ### Troubleshooting Mutagen Sync Issues
 
+    * Does `ddev start` take forever, with perhaps many minutes of "dots"? Take care that you don't have big binaries that mutagen is trying to sync, which will cause endless "dots" as your project is starting. There is already an exclusion place for database tarballs and the like, the `.tarballs` directory; if you put dumps there they will be ignored by mutagen. To find out what mutagen is trying to sync, use `ddev mutagen status -l` in another window.
```

Comment:
Should this be first? It seems like the existing items are probably more urgent to convey sooner than this.

Rewrite to start with a recommendation instead of a question, and try to be more brief:

```suggestion
    * Avoid having Mutagen sync large binaries, which can cause `ddev` start to take a long time. The `.tarballs` directory is automatically excluded, so Mutagen will ignore anything you move there. To see what it’s trying to sync, run `ddev mutagen status -l` in another window.
```